.github/workflows/documentation.yml

```diff
@@ -0,0 +1,35 @@
+name: DOCUMENTATION
+
+on: [push]
+
+jobs:
+
+  documentation:
+    runs-on: espressopp/buildenv:ubuntu
+
+    steps:
+    - name: build documentation
+      run: |
+        git clone https://github.com/espressopp/espressopp.git
```

Comment:
Use the checkout action instead.

Reply:
Same as for docker containers. Do you want to publish a documentation for every WIP branch?

Reply:
We want to test if the documentation still builds before it gets merged.